astropy/io/misc/asdf/tags/transform/tests/test_transform.py

```diff
@@ -180,52 +181,6 @@ def test_bounding_box(tmpdir):
     helpers.assert_roundtrip_tree(tree, tmpdir)
 
 
-@pytest.mark.parametrize("standard_version", asdf.versioning.supported_versions)
```

Comment:
This test is covering a difference in behavior between the supported ASDF Standard versions, I think we should keep it.